.changeset/gorgeous-bears-glow.md

```diff
@@ -0,0 +1,63 @@
+---
+'braid-design-system': major
+---
+
+Improved trimming of white space around text
```

Comment:
```suggestion
**`apac` and `seekBusiness` themes:** Improved trimming of white space around text
```

Reply:
Ignore me, I forgot that this also changes the height of text elements in the other themes.